docs/developer-guide/ksqldb-clients/java-client.md

```diff
@@ -680,6 +680,25 @@ topic name, column names and associated types, serialization formats, queries th
 read and write from the stream or table, and more. For more details, see the
 [API reference](api/io/confluent/ksql/api/client/Client.html#describeSource(java.lang.String)).
 
+Connect to a {{ site.ccloud }} ksqlDB cluster <a name="connect-to-cloud"></a>
+-----------------------------------------------------------------------------
+
```

Comment:
Can we link to this section after the sentence that says "For additional client options, see the API reference." above? I think that's a more natural starting point for new users of the client. After they see that `ClientOptions` exists, then they'll wonder about the specific options required to connect to CCloud.